### DISCUSSION TITLE
Able to select date picker icon only in firefox but not in chrome.

### DISCUSSION THREAD

**Discussion by kharabela** [tailwindlabs/tailwindcss#13038] [Help] [answered]
https://play.tailwindcss.com/HP65MREGx0

1.Why?
2.What can I do to fix this?

**Comment by wongjn** [comment, marked as answer]
1. It seems like the container for the tooltip can overlap the icon, depending on the viewport size:
   ![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/68628c91-5da3-4f2a-8d30-93123d869ab0)
2. Slightly rework the tooltip layout so that 1 does not happen: https://play.tailwindcss.com/P3ac37ktJC
